corehq/apps/hqcase/views.py

```diff
@@ -175,6 +180,15 @@ def _handle_case_update(request, is_creation, case_id=None):
     if not is_creation and case_id and 'case_id' not in data:
         data['case_id'] = case_id
 
+    try:
+        if isinstance(data, list):
```

Comment:
@zandre-eng When will this be a list?

Reply:
Both single and bulk PUT/POST requests go through `_handle_case_update()`, so it will be a list whenever a bulk create or update request is done.